debezium-core/src/main/java/io/debezium/text/Position.java

```diff
@@ -65,6 +65,11 @@ public int line() {
         return line;
     }
 
+    @Override
```

Comment:
Is this legit? Seems equals and hashCode might be out of sync?

Reply:
The original source did not have an `#equals` at all and Checkstyle 8.19 detected this as a problem.  My goal wasn't to change behavior and only code formatting hence why I added the method this way.  I can certainly sync the two, though I think I'd rather see that as a separate issue.

Reply:
Ok, sounds good. In fact, I'm doubting that the class is even used at all. It originates from the legacy parser IIRC, so it might just be removed altogether. Can you file an issue for tracking this? Thanks!